src/extension/api/graphql.ts

```diff
@@ -1,27 +1,11 @@
 import { gql } from '@apollo/client'
 
-export interface CellExecutionMetadata {
-  mimeType: string
-  category: string
-  name: string
-}
+import { MutationCreateCellExecutionArgs } from '../__generated__/graphql'
 
-export interface CellExecutionRequest {
-  stdout: any
-  input: any
-  exitCode: number
-  pid: number
-  metadata: CellExecutionMetadata
-}
+export const createCellExecutionQuery = ({ data }: MutationCreateCellExecutionArgs) => {
+  const { input, exitCode, stdout, pid, metadata } = data
```

Comment:
One concern I wonder about: What if the names on one of these properties changes? Then the name will change for the property in `data`, but we might miss changing it in the mutation call, which is not typesafe.

Do you know if it possible here to pass the `data` object directly to the mutation?

Reply:
With the new changes, this file was completely removed!